[2023-02-21T11:53:51.007-0600] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 3_retries.task_c manual__2023-02-21T17:53:49.337492+00:00 [queued]>
[2023-02-21T11:53:51.018-0600] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 3_retries.task_c manual__2023-02-21T17:53:49.337492+00:00 [queued]>
[2023-02-21T11:53:51.018-0600] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-02-21T11:53:51.019-0600] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-02-21T11:53:51.019-0600] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-02-21T11:53:51.027-0600] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): task_c> on 2023-02-21 17:53:49.337492+00:00
[2023-02-21T11:53:51.031-0600] {standard_task_runner.py:54} INFO - Started process 525769 to run task
[2023-02-21T11:53:51.033-0600] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '3_retries', 'task_c', 'manual__2023-02-21T17:53:49.337492+00:00', '--job-id', '660', '--raw', '--subdir', 'DAGS_FOLDER/Apache_Airflow-The_Operators_Guide/3_retries.py', '--cfg-path', '/tmp/tmptz0u13p3']
[2023-02-21T11:53:51.034-0600] {standard_task_runner.py:83} INFO - Job 660: Subtask task_c
[2023-02-21T11:53:51.035-0600] {dagbag.py:525} INFO - Filling up the DagBag from /c/Airflow/dags/Apache_Airflow-The_Operators_Guide/3_retries.py
[2023-02-21T11:53:51.125-0600] {task_command.py:384} INFO - Running <TaskInstance: 3_retries.task_c manual__2023-02-21T17:53:49.337492+00:00 [running]> on host Tinmar.
[2023-02-21T11:53:51.175-0600] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Armando
AIRFLOW_CTX_DAG_ID=3_retries
AIRFLOW_CTX_TASK_ID=task_c
AIRFLOW_CTX_EXECUTION_DATE=2023-02-21T17:53:49.337492+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-21T17:53:49.337492+00:00
[2023-02-21T11:53:51.176-0600] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-02-21T11:53:51.177-0600] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'task_c'"]
[2023-02-21T11:53:51.183-0600] {subprocess.py:86} INFO - Output:
[2023-02-21T11:53:51.184-0600] {subprocess.py:93} INFO - task_c
[2023-02-21T11:53:51.188-0600] {subprocess.py:97} INFO - Command exited with return code 0
[2023-02-21T11:53:51.211-0600] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=3_retries, task_id=task_c, execution_date=20230221T175349, start_date=20230221T175351, end_date=20230221T175351
[2023-02-21T11:53:51.245-0600] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-02-21T11:53:51.263-0600] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
